packages/web/components/Landing/sections/Cards.tsx

```diff
@@ -0,0 +1,119 @@
+import { Box, Flex, Image, ListItem, OrderedList, Text } from '@metafam/ds';
+import CardBackground from 'assets/landing/card-background.png';
+import CardImage from 'assets/landing/card-image.png';
+
+export const Cards: React.FC = () => (
+  <Box
+    width="100%"
+    maxHeight="100%"
+    background="dark"
+    bgPosition="center"
+    bgSize="cover"
+  >
+    <Flex spacing={0} direction={{ base: 'column', md: 'row' }}>
+      <Box
+        backgroundImage={`url(${CardBackground})`}
+        bgPosition="center"
+        bgSize="cover"
+        width={{ base: '100%', md: '33%' }}
+        display="flex"
+        alignItems="center"
+        flexDirection="column"
+        pl="76px"
```

Comment:
on this page there are many paddings, widths, and particularly fontSize is in pxs. We want to replace them with rem equivalents and the fontSize to sm, md, lg or xl as necessary

Reply:
perfect I've fixed all of these to rem. Apologies I missed out on those others.